packages/manager/modules/pci/src/projects/project/storages/databases/database/advanced-configuration/advanced-configuration.scss

```diff
@@ -1,5 +1,5 @@
 ovh-manager-pci-project-database-advanced-configuration {
-  .ui-select-dropdown {
-    z-index: 1021 !important;
+  .advanced-configuration-json {
+    z-index: 1009 !important;
```

Comment:
Why `1009` ?

Reply:
Having values above `1009` causes the languages dropdown to be hidden by `the advanced-configuration-json ` part: 

With `z-index: 1010`
![image](https://user-images.githubusercontent.com/24541481/165041807-5b0c53b7-6523-4355-9d0b-83027c5caaaa.png)

With `z-index: 1009`
![image](https://user-images.githubusercontent.com/24541481/165041924-9e4cd515-e91d-420f-b0ec-b2e32e9108b7.png)